io_test.go

```diff
@@ -221,8 +340,13 @@ func TestOutput(t *testing.T) {
 			cfg.RegularGrid(ctmdata, pop, popIndices, mr, emis),
 		},
 		CleanupFuncs: []DomainManipulator{
-			Output(TestOutputFilename, false, "TotalPop deaths", "TotalPop",
-				"TotalPM25", "PM2.5 emissions", "Baseline TotalPM25", "WindSpeed"),
+			Output(TestOutputFilename, false, map[string]string{
+				"TotalPopDeaths":    "coxHazard(loglogRR(TotalPM25), TotalPop, MortalityRate)",
+				"TotalPop":          "TotalPop",
+				"TotalPM25":         "TotalPM25",
+				"PM25Emissions":     "PM25Emissions",
+				"BaselineTotalPM25": "BaselineTotalPM25",
```

Comment:
I think what we want to do here is, instead of letting the shapefile library truncate the output variable name if it is more than 10 characters, to add a check somewhere that returns an error if the output name is more than 10 characters. Then we can change this, for example to:

 "BasePM25": "BaselineTotalPM25",